stix_shifter/stix_translation/src/modules/security_advisor/json/to_stix_map.json

```diff
@@ -0,0 +1,80 @@
+{
+
+  "author_id": {
+    "key": "author.user_id"
```

Comment:
This should be mapped to `user-account.user_id`

Reply:
Done